external/emulation/emulation/_emulate/microphysics.py

```diff
@@ -54,17 +76,21 @@ def predict(self, x):
 @print_errors
 def _load_tf_model(model_path: str) -> tf.keras.Model:
     logger.info(f"Loading keras model: {model_path}")
-    with get_dir(model_path) as local_model_path:
-        model = tf.keras.models.load_model(local_model_path)
-        model = RenamedOutputModel(
-            model,
-            # for backwards compatibility
-            translation={
-                "air_temperature_output": "air_temperature_after_precpd",
-                "specific_humidity_output": "specific_humidity_after_precpd",
-                "cloud_water_mixing_ratio_output": "cloud_water_mixing_ratio_after_precpd",  # noqa: E501
-            },
-        )
+
+    if model_path == "NO_MODEL":
```

Comment:
Would be cleaner IMO to make this the default behavior of `prognostic_run.py` without a `--model` argument rather than specifying the magic NO_MODEL string.